src/Common/nunit/XmlHelper.cs

```diff
@@ -166,10 +165,9 @@ public static DateTime GetAttribute(this XmlNode result, string name, DateTime d
                 return defaultValue;
 
             DateTime date;
-            if (!DateTime.TryParse(dateStr, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal | DateTimeStyles.AllowWhiteSpaces, out date))
-                return defaultValue;
-
-            return date;
+            return !DateTime.TryParse(dateStr, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal | DateTimeStyles.AllowWhiteSpaces, out date) 
+                ? defaultValue 
+                : date;
```

Comment:
That's a little flip. Should one programmer change anothers work stylistically for no other reason than the fact that they can?